spec/services/proofing/resolution/progressive_proofer_spec.rb

```diff
@@ -95,363 +131,387 @@
-        allow(instant_verify_proofer).to receive(:proof).
-          and_return(residential_instant_verify_proof)
-        allow(residential_instant_verify_proof).to receive(:success?).and_return(true)
-      end
 
-      it 'only makes one request to LexisNexis InstantVerify' do
-        expect(instant_verify_proofer).to receive(:proof).exactly(:once)
-        expect(aamva_proofer).to receive(:proof)
+      it 'returns a ResultAdjudicator' do
+        proofing_result = proof
 
-        subject
+        expect(proofing_result).to be_an_instance_of(Proofing::Resolution::ResultAdjudicator)
+        expect(proofing_result.same_address_as_id).to eq(applicant_pii[:same_address_as_id])
       end
 
-      it 'produces a result adjudicator with correct information' do
-        expect(aamva_proofer).to receive(:proof)
-
-        result = subject
-
-        expect(result.same_address_as_id).to eq('true')
-        expect(result.ipp_enrollment_in_progress).to eq(true)
-        expect(result.double_address_verification).to eq(true)
-        expect(result.resolution_result).to eq(result.residential_resolution_result)
-      end
-
-      context 'LexisNexis InstantVerify fails' do
-        let(:result_that_failed_instant_verify) do
+      context 'residential address and id address are the same' do
+        let(:aamva_proofer) { instance_double(Proofing::Aamva::Proofer) }
+        let(:residential_instant_verify_proof) do
           instance_double(Proofing::Resolution::Result)
         end
         before do
-          allow(instance).to receive(:proof_id_address_with_lexis_nexis_if_needed).
-            and_return(result_that_failed_instant_verify)
-          allow(instant_verify_proofer).to receive(:proof).with(hash_including(state_id_address)).
-            and_return(result_that_failed_instant_verify)
-          allow(instance).to receive(:user_can_pass_after_state_id_check?).
-            with(result_that_failed_instant_verify).
-            and_return(true)
-          allow(result_that_failed_instant_verify).to receive(:success?).
-            and_return(false)
+          allow(instance).to receive(:with_state_id_address).and_return(transformed_pii)
+          allow(instance).to receive(:state_id_proofer).and_return(aamva_proofer)
+          allow(instance).to receive(:resolution_proofer).and_return(instant_verify_proofer)
+          allow(instant_verify_proofer).to receive(:proof).
+            and_return(residential_instant_verify_proof)
+          allow(residential_instant_verify_proof).to receive(:success?).and_return(true)
         end
 
-        context 'the failure can be covered by AAMVA' do
-          before do
-            allow(result_that_failed_instant_verify).
-              to receive(:attributes_requiring_additional_verification).
-              and_return([:address])
-          end
+        it 'only makes one request to LexisNexis InstantVerify' do
```

Comment:
oooo this was def a gap, thanks for adding coverage for this!

Reply:
So thankfully this one was already in there I think.

Reply:
oh i must have missed it before!